src/simulation/dynamics/Thrusters/thrusterStateEffector/thrusterStateEffector.rst

```diff
@@ -54,14 +54,23 @@ While the value of :math:`\kappa_i` for each thruster is computed numerically by
 
 where :math:`\kappa_{0,i}` corresponds to the initial conditions of the ``thrusterFactor`` variable for the i-th particular thruster.
 
-If the thruster is added to an auxiliary body :math:`F` connect to the main body :math:`B`, then this is accomplished
-using::
+If the thruster is added to an auxiliary body :math:`F`, of type :ref:`prescribedMotionStateEffector`,
+which is connected to the main spacecraft body :math:`B`, then this is accomplished using::
 
     thrusterSet.addThruster(thruster1, bodyStatesMsg)
 
-where ``thruster1`` is the thruster being added, and ``bodyStatesMsg`` is the state output message of
+Here ``thruster1`` is the thruster being added, and ``bodyStatesMsg`` is the state output message of
 the platform :math:`F`.  In the ``Update()`` routine the position of the platform :math:`F` relative
-to math:`B` is recomputed each time step.  If the thruster is added
+to :math:`B` is recomputed each time step.
+
+If the thruster is attached to the hub, then all output message variables that
+contain ``_B`` are written in the spacecraft hub :math:`B` frame as usual.
+However, if the thruster is attached to a :ref:`prescribedMotionStateEffector`,
```

Comment:
I'm not sure why we're talking specifically about a prescribed body here. I'm assuming it would be the same issue even when the thruster is attached to a generic body other than spacecraft, right? At least in my case, I'm not using a prescribed body.

Reply:
@rcalaon , you are correct.  We should discuss what is happening here and clarify.  We can add this to hingedRigidBody, for example, but this only determines the orientation and location of the thruster.  The thruster does not impact, currently, the hingedRigidBody dynamics.

Reply:
@rcalaon and @joaogvcarneiro , I just pushed suggested edits to address Riccardo's feedback.  I rewrote much of this section, and added sub-section titles to break up the text better.  Let me know if you like these new text now.